src/Cli/Program.cs

```diff
@@ -129,7 +129,12 @@ public static int Main(string[] args)
                         {
                             try
                             {
-                                HttpClient client = new() { BaseAddress = new Uri("http://localhost:5000/graphql") };
+                                // CodeQL[SM02185] Loading internal server connection
```

Comment:
For this suppression to be honored, this probably needs to be on the same line which is detected as an alert.

Reply:
Giving that a try now. I'm use to JS where the comment applies to the next line